src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -1616,9 +1616,13 @@ Expression RegularCondition():
 	| "<" { result = new MinorThan(); }
 	| "=" { result = new EqualsTo(); }
 	| ">=" { result = new GreaterThanEquals(); }
+	| "> =" { result = new GreaterThanEquals("> ="); }
 	| "<=" { result = new MinorThanEquals(); }
+	| "< =" { result = new MinorThanEquals("< ="); }
 	| "<>" { result = new NotEqualsTo(); }
+	| "< >" { result = new NotEqualsTo("< >"); }
```

Comment:
Isnt this only half the way? Oracle allows, as I learned right now :), multiple whitespaces between < and =. Addionaly wouldnt it be better to map <..= to <=? Is there a case one need to reproduce oracles special space separated syntax?